packages/fluentui/react-builder/src/components/Canvas.tsx

```diff
@@ -238,6 +341,12 @@ export const Canvas: React.FunctionComponent<CanvasProps> = ({
             outline-offset: -1px;
           }
           `,
+        inUseMode &&
```

Comment:
When I use the virtual cursor and later disable it but stay in the use mode, the virtual cursor frame remains rendered. I think it should be removed.